aws-lc-rs/src/cmac.rs

```diff
@@ -586,4 +664,95 @@ mod tests {
         assert_eq!(tag.as_ref().len(), 8); // 3DES block size
         assert!(verify(&key, data, tag.as_ref()).is_ok());
     }
+
+    #[test]
+    fn cmac_sign_to_buffer_test() {
```

Comment:
why put these tests here instead of cmac_tests.rs? doesn't look like they directly call any crate-internal functions